exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/parser/DrillCompoundIdentifier.java

```diff
@@ -69,31 +70,38 @@ public void addIndex(int index, SqlParserPos pos){
     }
   }
 
-  public SqlNode getAsSqlNode(){
-    if(ids.size() == 1){
+  public SqlNode getAsSqlNode(Set<DrillCompoundIdentifier> fullSchemasSet) {
```

Comment:
I agree that this current bracketless Syntax for support complex queries  and converting of original sqlNode from calcite must be improve in future but not sure how it is possible now. I think a new approach could be after resolving [CALCITE-1208](https://issues.apache.org/jira/browse/CALCITE-1208)
I added TODO for this.